cli-feed-v3-3.json

```diff
@@ -0,0 +1,269 @@
+            "projectTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ProjectTemplates/2.1.1642",
+            "projectTemplateId": {
+              "csharp": "Microsoft.AzureFunctions.ProjectTemplate.CSharp.2.x"
+            },
+            "localContainerBaseImage": "DOCKER|mcr.microsoft.com/azure-functions/dotnet:2.0-appservice",
+            "serviceAppSettings": {
+              "FUNCTIONS_EXTENSION_VERSION": "~2",
+              "FUNCTIONS_WORKER_RUNTIME": "dotnet"
+            },
+            "windowsSiteConfig": {
+              "netFrameworkVersion": "v4.0"
+            },
+            "linuxSiteConfig": {
+              "linuxFxVersion": "DOTNET|2.2"
+            }
+          }
+        }
+      },
+      "coreTools": [
+        {
+          "OS": "Linux",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.7.3188/Azure.Functions.Cli.linux-x64.2.7.3188.zip",
+          "sha2": "f8cff611d0cc2198589a7ff0b1b268549bf86c11afd799098df59f7721d6b771",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "MacOS",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.7.3188/Azure.Functions.Cli.osx-x64.2.7.3188.zip",
+          "sha2": "baa850c449e9be5fcef8e2bf91883885ec9ab9471437c998701435f5164d72e5",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.7.3188/Azure.Functions.Cli.min.win-x64.2.7.3188.zip",
+          "sha2": "1c5c49bfb683e45fc4b44e52e25f63324f9d8e931d47259c55fc66c9b7019fc9",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.7.3188/Azure.Functions.Cli.win-x86.2.7.3188.zip",
+          "sha2": "4a0342497a606e763aa148a10852d6da91d2d5cc7fdb7e705a0c17df4e038023",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.7.3188/Azure.Functions.Cli.min.win-x86.2.7.3188.zip",
+          "sha2": "f04accdc23432695786f434835b12558e603866a5c3b6d5c398965d3dccc78e9",
+          "size": "minified",
+          "default": "true"
+        }
+      ]
+    },
+    "3.20.0": {
+      "templates": "https://functionscdn.azureedge.net/public/TemplatesApi/3.1.1648.zip",
+      "workerRuntimeInformation": {
+        "dotnet": {
+          "net5-isolated": {
+            "displayInfo": {
+              "displayName": ".NET 5 (Isolated)",
+              "hidden": true,
+              "diplayVersion": "v3",
+              "targetFramework": ".NET 5"
+            },
+            "capabilites": "outOfProc,net5",
```

Comment:
Can we be consistent and use "isolated" instead of "outOfProc" in all cases?

Also, I think it's fair to assume it's "inProc" unless "isolated" is specified, so I don't know that we even need to specify "inProc" anywhere. Similar to how `FUNCTIONS_WORKER_RUNTIME` is `dotnet` vs. `dotnet-isolated`, not `dotnet-inProc` vs. `dotnet-outOfProc`

Reply:
Yeah that makes sense.